administration/src/bp-modules/applications/ApplicationsOverview.tsx

```diff
@@ -137,17 +160,30 @@ const ApplicationView: FunctionComponent<{
               justifyContent: 'space-between',
               marginTop: '20px',
             }}>
-            {height > COLLAPSED_HEIGHT ? (
-              <PrintAwareButton onClick={() => setCollapsed(true)} icon='caret-up'>
-                Weniger anzeigen
+            <ButtonGroup fill alignText='left'>
+              {height > COLLAPSED_HEIGHT ? (
+                <PrintAwareButton onClick={() => setCollapsed(true)} icon='caret-up'>
+                  Weniger anzeigen
+                </PrintAwareButton>
+              ) : null}
+              <PrintAwareButton onClick={() => printApplicationById(id)} intent='primary' icon='print'>
+                PDF exportieren
+              </PrintAwareButton>
+              <Tooltip
```

Comment:
in which case will this happen? If we show this tooltip it should explain why this can't be done.
Shouldn't all applications have sufficient information?
I would either explain why the card can't be created from application or even not show the button at all

Reply:
This was mainly a follow up on the remark @michael-markl made. Whe an application cannot be mapped to a card then this tooltip is displayed and the button is disabled. I'll thik about how the text could be improved :). Currently(!) this should never be the case. But what I definitely forgot was to not shouw this button for Nürnberg. I'll add this.